Readme.md

```diff
@@ -59,6 +59,33 @@ Similarly, you can execute `npm run dotnet-postclean --if-present` via the `dotn
 }
 ```
 
+## Copying vendor libraries into _wwwroot_
+
+If you intend to do something more advanced with Node.js dependencies, you may want to use our [Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions) project instead. But if you just want to fetch vendor packages from NPM and copy them into _./wwwroot/vendors_ directory, follow these steps:
+
+1. Make sure the _package.json_ file contains the desired NPM packages in the `dependencies` or `devDependencies` section. For example:
+   ```json
+   {
+     "private": true,
+     "devDependencies": {
+       "chart.js": "4.4.9",
+       "chartjs-plugin-annotation": "3.1.0",
+       "chartjs-plugin-datalabels": "2.2.0"
+     }
+   }
+   ```
+2. In your _.csproj_ file, find an `<ItemGroup>` and add `<Vendor Include="node_modules/{package_name}" />` items for each.
+   - If the distribution files are located in a _dist_ subdirectory of the package (e.g. _./node_modules/chart.js/dist_), then you are good to go.
+   - Otherwise, include a child element `<Subdirectory>some/other/path</Subdirectory>` 
+     ```xml
+         <Vendor Include="node_modules/chart.js">
+           <Subdirectory>dist</Subdirectory>
+         </Vendor>
+     ```
+   - Even if you want to copy the whole package directory, include an empty `<Subdirectory />` element, otherwise you'll get a warning.
```

Comment:
Not just this? "Even" confused me that I need to use `Subdirectory˙˙ always.

```suggestion
   - If you want to copy the whole package directory, include an empty `<Subdirectory />` element, otherwise you'll get a warning.
```